src/externalized/AmmoTypeModel.cc

```diff
@@ -18,26 +18,26 @@ AmmoTypeModel::~AmmoTypeModel() {}
 
 void AmmoTypeModel::serializeTo(JsonObject &obj) const
 {
-  obj.AddMember("index",                index);
-  obj.AddMember("internalName",         internalName.c_str());
+	obj.AddMember("index",                index);
+	obj.AddMember("internalName",         internalName.c_str());
 }
 
 AmmoTypeModel* AmmoTypeModel::deserialize(JsonObjectReader &obj)
 {
-  int index = obj.GetInt("index");
-  const char *internalName = obj.GetString("internalName");
-  return new AmmoTypeModel(index, internalName);
+	int index = obj.GetInt("index");
+	const char *internalName = obj.GetString("internalName");
+	return new AmmoTypeModel(index, internalName);
 }
 
 
 const AmmoTypeModel* getAmmoType(const char *ammoTypeName,
-                                 const std::map<std::string, const AmmoTypeModel*> &ammoTypeMap)
+					const std::map<std::string, const AmmoTypeModel*> &ammoTypeMap)
```

Comment:
Are you sure? I think that's very confusing if the template arguments are split up.